src/rpdk/core/jsonutils/resolver.py

```diff
@@ -7,6 +7,7 @@
 
 UNDEFINED = "undefined"
 MULTIPLE = "multiple"
+DEFAULT = "default"
```

Comment:
`default` is a reserved name in schema engine, could we use some alternatives as it's a bit confusing

Reply:
`default` is still a good name as long as it's allowed to use.  Maybe the constant name should be changed to more meaningful, such as `FORMAT_DEFAULT`

Reply:
i can change this variable name. "default" is just used in the mapping of resolved schema types to language primitives, so it wouldn't be something that the user would specify in the schema:
https://github.com/jotompki/cloudformation-cli-java-plugin/blob/d006d5d22afd455d8146d375e6438c82644b3e45/python/rpdk/java/resolver.py#L6-L14

Either way, `default` is a reserved key, whereas this would be a value for the `format` key, so if a user did want to explicitly specify the default for the languages, they would do the following:
```
{
    "type": "integer",
    "format": "default"
}
```